internal/provider/model/job_definition/input_option/mysql.go

```diff
@@ -116,7 +116,7 @@ func (mysqlInputOption *MySQLInputOption) ToUpdateInput() *input_options2.Update
 	}
 }
 
-func toInputOptionColumnsInput(columns []InputOptionColumn) []input_options2.InputOptionColumn {
+func toMysqlInputOptionColumnsInput(columns []InputOptionColumn) []input_options2.InputOptionColumn {
```

Comment:
NITS: Since this function is not used externally, rename is not necessary.

Reply:
I agree, but I changed the input_option_column in mysql and snowflake to identify it.